reactos/drivers/usb/usbport/usbport.h

```diff
@@ -0,0 +1,1129 @@
+#ifndef USBPORT_H__
+#define USBPORT_H__
+
+#include <ntddk.h>
+#include <windef.h>
+#include <stdio.h>
+#include <wdm.h>
+#include <wdmguid.h>
+#include <usb.h>
+#include <hubbusif.h>
+#include <usbbusif.h>
+#include <usbdlib.h>
+#include <usbuser.h>
+#include "..\usbmport.h"
+
+#define PCI_INTERFACE_USB_ID_UHCI 0x00
+#define PCI_INTERFACE_USB_ID_OHCI 0x10
+#define PCI_INTERFACE_USB_ID_EHCI 0x20
+#define PCI_INTERFACE_USB_ID_XHCI 0x30
+
+#ifdef USBD_TRANSFER_DIRECTION
```

Comment:
usb.h:
#define USBD_TRANSFER_DIRECTION               0x00000001

but RosBE say:
../drivers/usb/usbport/device.c: In function 'USBPORT_SendSetupPacket':
../drivers/usb/usbport/device.c:58:49: error: 'USBD_TRANSFER_DIRECTION' undeclar
ed (first use in this function)